packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1115,4 +1117,265 @@ describe('KeyringController', () => {
       expect(cancelSignRequestStub.called).toBe(true);
     });
   });
+
+  describe('Ledger Keyring', () => {
```

Comment:
Are there any other methods that we need to test here? I notice these new methods that don't seem to be reflected here:

- `connectLedgerHardware`
- `openEthereumApp`
- `closeRunningApp`

I see that there was discussion in #862 around this toward the end, did you feel like it wasn't worth adding these?

Also, there are state changes that `unlockLedgerDefaultAccount` makes — it looks like those aren't tested either.